mep/common/templates/common/widgets/rangewidget.html

```diff
@@ -1,4 +1,4 @@
 {% comment %}
 Template for RangeWidget with numeric start and stop inputs.
 {% endcomment %}
-<div class="inputs">{% spaceless %}{% for subwidget in widget.subwidgets %}{% include subwidget.template_name with widget=subwidget %}{% if forloop.first %}<span class="separator">to</span>{% endif %}{% endfor %}{% endspaceless %}</div>
+<div class="inputs">{% spaceless %}{% for subwidget in widget.subwidgets %}{% include subwidget.template_name with widget=subwidget %}{% if forloop.first %}<span class="separator" aria-label="to">–</span>{% endif %}{% endfor %}{% endspaceless %}</div>
```

Comment:
`aria-label` is great here

Reply:
ended up changing all of them to use `<span class="sr-only">` because it's more concise